src/pages/regions/index.tsx

```diff
@@ -21,6 +21,7 @@ const RegionsPage: FC<Props> = ({
     regions: { nodes: regions },
   },
 }) => {
+  console.log('dataatatatata', regions)
   const createRegionHref = (region: Region) => {
```

Comment:
suggestion: use [gatsbyPath](https://www.gatsbyjs.com/docs/reference/routing/file-system-route-api/#gatsbypath-example) approach to making sure the you get correct slugified path